File: GetStartedCalls-Start/app/src/main/java/com/nexmo/client/getstarted/calls/LoginActivity.java
package com.nexmo.client.getstarted.calls;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.nexmo.client.NexmoClient;
import com.nexmo.client.NexmoUser;
import com.nexmo.client.request_listener.NexmoApiError;
import com.nexmo.client.request_listener.NexmoRequestListener;

public class LoginActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        NexmoHelper.init(getApplicationContext());
    }

    public void onLoginJaneClick(View view) {
        loginToSdk(NexmoHelper.JWT_JANE);
    }

    public void onLoginJoeClick(View view) {
        loginToSdk(NexmoHelper.JWT_JOE);
    }

    private void loginToSdk(String token) {
        //TODO: to complete
    }
}

File: GetStartedCalls-Start/app/src/main/java/com/nexmo/client/getstarted/calls/LoginActivity.kt
package com.nexmo.client.getstarted.calls

import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_login.*

class LoginActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        btnLoginJoe.visibility =
                if (enabledFeatures.contains(Features.IN_APP_to_IN_APP)) {
                    View.VISIBLE
                } else
                    View.GONE

        init(applicationContext)
    }

    fun onLoginJaneClick(view: View) {
        loginToSdk(JWT_JANE)
    }

    fun onLoginJoeClick(view: View) {
        loginToSdk(JWT_JOE)
    }

    private fun loginToSdk(token: String) {
        //TODO: to complete
    }
}
